package.json

```diff
@@ -16,7 +16,9 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "generate": "turbo run generate",
     "prepare": "husky install",
-    "github:release": "pnpm changeset tag && git push --follow-tags"
+    "github:release": "pnpm changeset tag && git push --follow-tags",
+    "check-deps": "syncpack fix-mismatches --source \"apps/*/package.json\" --source \"packages/*/package.json\" --filter \"@saleor/app-sdk|@saleor/macaw-ui\"",
+    "postinstall": "pnpm check-deps"
```

Comment:
@peelar I think it should not be here, it runs on vercel and maybe it fails builds.

We can add hooks in next PRs (recommit instead of running on entire codebase)